web/packages/teleterm/src/ui/ConnectMyComputer/connectMyComputerContext.test.tsx

```diff
@@ -225,3 +225,58 @@ test('starts the agent automatically if the workspace autoStart flag is true', a
   ).toHaveBeenCalledTimes(1);
   expect(appContext.connectMyComputerService.runAgent).toHaveBeenCalledTimes(1);
 });
+
+describe('canUse returns true', () => {
+  test('if the user has permissions', async () => {
+    const { appContext, rootCluster } = getMocksWithConnectMyComputerEnabled();
+    const isAgentConfigFileCreated = Promise.resolve(false);
+    jest
+      .spyOn(appContext.connectMyComputerService, 'isAgentConfigFileCreated')
+      .mockReturnValue(isAgentConfigFileCreated);
+
+    const { result } = renderHook(() => useConnectMyComputerContext(), {
+      wrapper: ({ children }) => (
+        <MockAppContextProvider appContext={appContext}>
+          <WorkspaceContextProvider value={null}>
+            <ConnectMyComputerContextProvider rootClusterUri={rootCluster.uri}>
+              {children}
+            </ConnectMyComputerContextProvider>
+          </WorkspaceContextProvider>
+        </MockAppContextProvider>
+      ),
+    });
+
+    await act(() => isAgentConfigFileCreated);
+
+    expect(result.current.canUse).toBeTruthy();
+  });
+
+  test('if the user does not have permissions, but the agent has been configured', async () => {
+    const { appContext, rootCluster } = getMocksWithConnectMyComputerEnabled();
+    // remove Connect My Computer permissions
+    appContext.clustersService.setState(draftState => {
+      draftState.clusters.get(rootCluster.uri).loggedInUser.acl.tokens.create =
+        false;
+    });
+    const isAgentConfigFileCreated = Promise.resolve(true);
+    jest
+      .spyOn(appContext.connectMyComputerService, 'isAgentConfigFileCreated')
+      .mockReturnValue(isAgentConfigFileCreated);
+
+    const { result } = renderHook(() => useConnectMyComputerContext(), {
+      wrapper: ({ children }) => (
+        <MockAppContextProvider appContext={appContext}>
+          <WorkspaceContextProvider value={null}>
+            <ConnectMyComputerContextProvider rootClusterUri={rootCluster.uri}>
+              {children}
+            </ConnectMyComputerContextProvider>
+          </WorkspaceContextProvider>
+        </MockAppContextProvider>
+      ),
+    });
+
+    await act(() => isAgentConfigFileCreated);
+
+    expect(result.current.canUse).toBeTruthy();
```

Comment:
just double checking this should be truthy (i'm used no permissions == denied)

Reply:
Yeah, it should be truthy, because users should still be able to control the agent that they set up (for example they may want to remove it completely).
But I think it the _falsy_ case should be covered in tests, so I added one.